decidim-core/app/cells/decidim/linked_resources_for/show.erb

```diff
@@ -0,0 +1,3 @@
+<% linked_resources.each do |klass, resources| %>
```

Comment:
I guess these are still pending:
![Related resources](https://user-images.githubusercontent.com/864340/194035565-535ff3a2-2c00-4490-9b1b-babfb917afb3.png)

Maybe mark them in the pending list?

Reply:
Not exactly, the content of these "included" blocks are lists of the respective elements. Such elements are located in their specific gems, so they will redesign when we face these modules.